core/string/ustring.h

```diff
@@ -302,11 +302,11 @@ class String {
 	String pad_zeros(int p_digits) const;
 	String trim_prefix(const String &p_prefix) const;
 	String trim_suffix(const String &p_suffix) const;
-	String lpad(int min_length, const String &character = " ") const;
-	String rpad(int min_length, const String &character = " ") const;
+	String lpad(int min_length, const String &p_character = " ") const;
+	String rpad(int min_length, const String &p_character = " ") const;
```

Comment:
Should add the `p_` prefix to both arguments.

```suggestion
	String lpad(int p_min_length, const String &p_character = " ") const;
	String rpad(int p_min_length, const String &p_character = " ") const;
```